oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -109,6 +109,62 @@ public class ComputeEngineCredentials extends GoogleCredentials
   static final int MAX_COMPUTE_PING_TRIES = 3;
   static final int COMPUTE_PING_CONNECTION_TIMEOUT_MS = 500;
 
+  // Experimental Feature:
+  //
+
+  /**
+   * Experimental Feature.
+   *
+   * <p>{@link GoogleAuthTransport} specifies how to authenticate to Google APIs.
+   *
+   * <p>Behavior of setting {@link GoogleAuthTransport} / {@link BindingEnforcement}:
+   *
+   * <p>MTLS-bound token where binding enforcement depends on IAM policy: MTLS / {}, {} /
+   * IAM_POLICY, MTLS / IAM_POLICY
+   *
+   * <p>MTLS-bound token where bindings are always enforced: {} / ON, MTLS / ON
+   *
+   * <p>DirectPath bound token: ALTS / {}
```

Comment:
Since this also shares logic with BindingEnforcement, thoughts on duplicating this in the javadocs for BindingsEnforcement as well?

Reply:
Or if there is a better way to split it up, I'm also fine with that

Reply:
Yes, duplicating this in `BindingEnforcement` javadocs SGTM. done.